sound-design.MD

```diff
@@ -0,0 +1,42 @@
+# MRE Sound API proposal
+
+This is node.js side only. It doesn't actually do anything yet.
+
+## Required functionality
+* preload sounds (as looping or one-shot)
+* play preloaded sounds
+* load and play sounds (Basically preload, instantly play, unload upon completion)
+* pause, resume, 
+* stop sounds 
+* set the sound state (to change volume, pitch, and where possible seek)
+
+## Design thoughts
+
+(Feel free to comment on each item separately)
+
+I'm using assetmanager to for the preloading sound to ask the client to load a sound resource, and return a handle. This is therefore are not instant, but return a promise.
+
+All sounds play on an actor, and the play calls therefore happen on the actor.
```

Comment:
>play calls therefore happen on the actor [](start = 37, length = 40)

Is there a limit to the number of sounds that can live on a single actor? Might get confusing routing commands to a particular instance in this case.

Reply:
doesn't seem like there is a limit. I don't think it's confusing as they are accessed from the instance, not from the actor.